docs/content/docs/tutorials/eventbridge-example.md

```diff
@@ -0,0 +1,308 @@
+---
+title: "Manage EventBridge event buses and rules with the ACK EventBridge Controller"
+description: "Send filtered events on a custom bus to SQS."
+lead: "Create and manage EventBridge event buses and rules directly from Kubernetes"
+draft: false
+menu:
+  docs:
+    parent: "tutorials"
+weight: 45
+toc: true
+---
+
+EventBridge is a serverless service that uses events to connect application components together, making it easier for
+you to build scalable event-driven applications. Use it to route events from sources such as home-grown applications,
+AWS services, and third- party software to consumer applications across your organization. EventBridge provides a simple
```

Comment:
```suggestion
AWS services, and third-party software to consumer applications across your organization. EventBridge provides a simple
```

Reply:
Funny, this is straight from our docs :D Good eye!